lib/services/sorting/post/post_score_functions.dart

```diff
@@ -4,16 +4,16 @@ import "package:proxima/models/database/post/post_firestore.dart";
 
 typedef PostScoreFunction = double Function(
   PostFirestore post,
-  GeoPoint position,
+  GeoPoint userPosition,
 );
 
 /// Increases by 1 every upvote, but decreases by 1 every day
-double hotScore(PostFirestore post, GeoPoint userPosition) {
-  return voteScore(post, userPosition) - dayScore(post, userPosition);
+double hotScore(PostFirestore post, GeoPoint _) {
+  return voteScore(post, _) - dayScore(post, _);
```

Comment:
The `_` notation is only useful for unused variable. Here the argument is actually used, so I think we should keep the original argument name.

Reply:
Fixed with [3e3a507](https://github.com/ProximaEPFL/proxima/pull/262/commits/3e3a5072f110a02550d1168e151453441530cb40)